x-pack/plugin/esql/qa/testFixtures/src/main/resources/eval.csv-spec

```diff
@@ -636,4 +636,55 @@ row foo = "Bar" | where foo rlike "(?i)Ba.*"
 foo:keyword
 ;
 
+evalAfterAvgGroupingUsingSameName
```

Comment:
Please also add the query from the [original issue](https://github.com/elastic/elasticsearch/issues/126026)